### PR TITLE
Add NO_SETPOINT_SET result to Offboard plugin

### LINKED ISSUE #732: Add `NO_SETPOINT_SET` To Offboard Results
`Offboard::result_str` and `OffboardImpl::offboard_result_from_command_result` are missing `NO_SETPOINT_SET`.

### PR DESCRIPTION
**Pull request by cswkim**
I think the 2 changed files are the only places that need updating, but I'm not 100% sure. If someone could please verify.

Resolves #732

### CODE REVIEW

File: plugins/offboard/offboard_impl.cpp [hunk 1]
```diff
@@ -459,6 +459,8 @@ Offboard::Result OffboardImpl::offboard_result_from_command_result(MAVLinkComman
             return Offboard::Result::COMMAND_DENIED;
         case MAVLinkCommands::Result::TIMEOUT:
             return Offboard::Result::TIMEOUT;
+        case MAVLinkCommands::Result::NO_SETPOINT_SET:
```
Comment (julianoes) [inline plugins/offboard/offboard_impl.cpp]: But `MAVLinkCommands::Result::NO_SETPOINT_SET` does not exist:
https://github.com/Dronecode/DronecodeSDK/blob/4332994c55c9ac8357954ccd1afcc5fa4a87172a/core/mavlink_commands.h#L19-L28

File: plugins/offboard/offboard.cpp [hunk 2]
```diff
@@ -67,6 +67,8 @@ const char *Offboard::result_str(Result result)
             return "Command denied";
         case Result::TIMEOUT:
             return "Timeout";
+        case Result::NO_SETPOINT_SET:
+            return "No setpoint set";
```
Comment (julianoes) [inline plugins/offboard/offboard.cpp]: This one is correct.

File: plugins/offboard/offboard_impl.cpp (hunk 1 above)
Comment (cswkim) [reply plugins/offboard/offboard_impl.cpp]: OK, just added.

File: core/mavlink_commands.h [hunk 3]
```diff
@@ -24,6 +24,7 @@ class MAVLinkCommands {
         COMMAND_DENIED,
         TIMEOUT,
         IN_PROGRESS,
+        NO_SETPOINT_SET,
```
Comment (julianoes) [inline core/mavlink_commands.h]: Nono, a MAVLink command can't ever have that result. It's an invention of the SDK :)

Comment (cswkim) [reply core/mavlink_commands.h]: Ah I see. OK I removed from mavlink command results and `offboard_result_from_command_result()`.

**Comment by JonasVautherin** [comment]
Yay! Thanks @cswkim!

**Pull request merged**